packages/web/src/scss/components/Button/_Button.scss

```diff
@@ -77,3 +77,23 @@
         theme.$border-width
     );
 }
+
+.Button--loading {
+    position: relative;
+    color: transparent;
+
+    &:hover,
+    &:active {
+        color: transparent;
+    }
+}
+
+.Button--loading > svg:last-child {
```

Comment:
And what if there was another svg here too?

Reply:
We have to hope. It's possible to control the order in the Twig or React implementation, not here in HTML. So at least add explicit mention about the order to the documentation. @crishpeen

Reply:
I guess this implementation is just fine. That's why there is the pseudo selector `last-child` that does the job, @tomassychra

Reply:
Exactly. The risk is only in the `web` package, where users has to place the Icon on the correct spot. I added warning to the Readme. In other packages we add the Spinner automatically so there is no problem :)